### DISCUSSION TITLE
Is it possible to configure prettier to have individual settings for blocks within a File

### DISCUSSION THREAD

**Discussion by IARI** [prettier/prettier#15536] [Q&A]
I'm working on a code base that has

- tabwidth 2 for html/XML and CSS content
- tabwidth 4 for js/ts content

consequently In vue SFC files there is tabwidth 2 in the template section, and tabwidth 4 in the script section. Is it possible to configure prettier to comply with this convention?

I know I can do [config overrides](https://prettier.io/docs/en/configuration#configuration-overrides), but I can only see that it is possible to specify filters for entire files there.

[here's also my identical stackoverflow post](https://stackoverflow.com/questions/77351507/is-it-possible-to-configure-prettier-to-have-individual-settings-for-vue-sfc-blo)

**Comment by kachkaev** [comment]
👋 @IARI! It’s a known problem, please follow the conversation in https://github.com/prettier/prettier/issues/5378.
